policyengine_us/variables/gov/states/ut/tax/income/credits/taxpayer_credit/ut_federal_deductions_for_taxpayer_credit.py

```diff
@@ -12,7 +12,20 @@ class ut_federal_deductions_for_taxpayer_credit(Variable):
     reference = "https://le.utah.gov/xcode/Title59/Chapter10/59-10-S114.html?v=C59-10-S114_2022032320220323"
 
     def formula(tax_unit, period, parameters):
-        us_itemizing = tax_unit("tax_unit_itemizes", period)
-        itm_ded_less_salt = tax_unit("itemized_deductions_less_salt", period)
-        std_ded = tax_unit("standard_deduction", period)
-        return where(us_itemizing, itm_ded_less_salt, std_ded)
+        p = parameters(period).gov.irs.deductions
```

Comment:
add comments for each line

Reply:
The following states that the filer can pick either the standard or the itemized deduction
you need 
```
        std_ded = tax_unit("standard_deduction", period)
        us_itemizing = tax_unit("tax_unit_itemizes", period)
```


![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/94dc8fa8-92de-4503-a8e0-d07fbcdb6cbb)

Reply:
please in the code, add line comments from this source